Tutorial 01: Click-by-click demonstration of worker area

pospec4444 opened the pull request:
Tell player about work area as requested in #3127.

- Demonstrate work area switch during lumberjack's hut construction.

pospec4444 committed:
Tutorial 01: Click-by-click demonstration of worker area

gunchleoc commented:
The instructions before the new task tell the player to close the building window. This instruction should be moved to "Waiting for the lumberjack to go up".

pospec4444 commented:
Sorry to say that, but I don't understand the review (again) :-(
(I was reading it several times for like five minutes.)

What do you want me to move and where to?
Do you want the player to close the building window manually instead of `close_windows()`?

gunchleoc commented:
When you place the tutorial, after placing the flag for the lumberjack, there is a message telling you to open the Lumberjack's window and to close it when you're done looking at it, followed immediately be the new message that you just added. So, telling the player to close the window doesn't make sense, because it needs to stay open for the demonstration of the working area.

pospec4444 committed:
Tutorial 01:
 - Changed texts about closing construction site window
 - Continue after player closes construction site window

pospec4444 commented:
Thanks for elaborating on the review. Let's hope that I got your idea :-)

Updated PR:
- Changed texts about closing construction site window
- Continue tutorial after player closes construction site window

________________________
If we want to put more accent on work areas, let's do it again in other tutorial. Naturally, it should be mentioned in Basic Controls because it **is** a basic control. On the other hand, work areas are crucial for some buildings (farms etc.) which are used later (Economy, Barbarians 01).

TL;DR: Shall we repeat work area in other tutorials?

pospec4444 committed:
Removed empty line

pospec4444 committed:
Proofreading of work area instructions

Co-Authored-By: gunchleoc <fios@foramnagaidhlig.net>

pospec4444 committed:
Proofreading of construction site instructions

Co-Authored-By: gunchleoc <fios@foramnagaidhlig.net>

pospec4444 committed:
Removed "close window" instructions.

pospec4444 commented:
> `When you have seen enough, close the construction site’s window...`

Removed.

klaus-halfmann commented:
Played the tutorial once again on this branch, fine for me. But I am not new to the game so I am not such a carefull tester.

Noordfrees reviewed (approved):
Played the tutorial with this branch, seems fine to me. I'll merge this now…
